test/e2e/cassandracluster_test.go

```diff
@@ -144,3 +150,124 @@ func cassandraClusterRollingRestartDCTest(t *testing.T, f *framework.Framework,
 
 	assert.Equal(t, false, cc.Spec.Topology.DC[0].Rack[0].RollingRestart)
 }
+
+func cassandraClusterInitMultiDCTest(t *testing.T, f *framework.Framework, ctx *framework.TestCtx) {
+	namespace, err := ctx.GetNamespace()
+	if err != nil{
+		t.Fatalf("Could not get namespace: %v", err)
+	}
+
+	cluster := &api.CassandraCluster{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "CassandraCluster",
+			APIVersion: "db.orange.com/v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "multidc-cluster",
+			Namespace: namespace,
+			Labels:    map[string]string{"cluster": "k8s.pic"},
+		},
+		Spec: api.CassandraClusterSpec{
+			BaseImage: "orangeopensource/cassandra-image",
+			NodesPerRacks: 2,
+			HardAntiAffinity: false,
+			DeletePVC: true,
+			Resources: api.CassandraResources{
+				Limits: api.CPUAndMem{
+					CPU: "500m",
+					Memory: "1Gi",
+				},
+			},
+			Topology: api.Topology{
+				DC: api.DCSlice{
+					api.DC{
+						Name: "dc1",
+						Rack: api.RackSlice{
+							api.Rack{
+								Name: "rack1",
+							},
+							api.Rack{
+								Name: "rack2",
+							},
+						},
+					},
+					api.DC{
+						Name: "dc2",
+						Rack: api.RackSlice{
+							api.Rack{
+								Name: "rack1",
+							},
+							api.Rack{
+								Name: "rack2",
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+
+
+
+	log.Debugf("Creating cluster")
+	if err := f.Client.Create(goctx.TODO(), cluster, cleanupOptions(ctx)); err != nil && !apierrors.IsAlreadyExists(err) {
+		t.Fatalf("Error Creating CassandraCluster: %v", err)
+	}
+
+	waitForClusterToBeReady(cluster, f, t)
+
+	services, err := f.KubeClient.CoreV1().Services(namespace).List(metav1.ListOptions{
+		LabelSelector: labels.FormatLabels(map[string]string{
+			"app": "cassandracluster",
+			"cassandracluster": cluster.Name,
+		}),
+	})
+	if err != nil {
+		t.Fatalf("Error listing services: %v", err)
+	}
+	assert.Equal(t, 2, len(services.Items))
+
+	// TODO add verification of services as well as statefulsets, etc.
+}
+
+func cleanupOptions(ctx *framework.TestCtx) *framework.CleanupOptions {
+	return NoCleanup()
+}
+
+func NoCleanup() *framework.CleanupOptions {
```

Comment:
It seems you only call that function in `cleanupOptions`, then why do you need it ?

Reply:
I added that in anticipation of needing `*framework.CleanupOptions` in multiple places. I wanted to have a single function that controls whether cleanup is performed for debugging; however, as I mentioned in my comment above, I am having trouble figuring out how to prevent resources from getting deleted after tests run.

Reply:
oh I see, so your PR is not ready yet for complete review. You should label it with `wip` then